components/dashboard/src/settings/selectClass.tsx

```diff
@@ -48,41 +48,24 @@ export default function SelectWorkspaceClass(props: SelectWorkspaceClassProps) {
                     Choose the workspace machine type for your workspaces.
```

Comment:
**suggestion**: I think this section can be safely moved further up just below the editor option, right? This would bring better visibility to this feature but also order preferences options more appropriately. What do you think? 

| BEFORE | AFTER |
|-|-|
| ![fo-settings-classes preview gitpod-dev com_preferences](https://user-images.githubusercontent.com/120486/179554130-67e08b0a-17c2-4278-aff3-5280c63db73e.png) | ![fo-settings-classes preview gitpod-dev com_preferences (1)](https://user-images.githubusercontent.com/120486/179554191-30c24390-fd8f-4ff7-9d3f-0a287c1a3db6.png) |